### PR TITLE
Revert "[nrf temphack] cmake: Add nRF specific hook"

### PR DESCRIPTION
**Pull request by carlescufi**
This reverts commit 86599ab896febfcdf9a58f6026f49e4a45c3c103.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

**Comment by carlescufi** [comment]
See https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/544

File: Jenkinsfile [hunk 1]
```diff
@@ -84,14 +83,6 @@ pipeline {
             }
           }
         }
-
-        stage('Build nrf samples') {
```
Comment (lemrey) [inline Jenkinsfile]: This was removed because it's now done by CI in `fw-nrfconnect-nrf`, correct?

Comment (carlescufi) [reply Jenkinsfile]: Correct

**Comment by carlescufi** [comment]
@lemrey @SebastianBoe the error in sanitycheck is the usual suspect. An unstable test. I need to find a way to exclude that board.

**Pull request merged**